lib/esv.rb

```diff
@@ -16,7 +16,27 @@ def self.generate_file(path, &block)
     end
   end
 
-  def self.parse(data)
+  # You can register your own header converters in this Hash.
+  #
+  # @see https://rubyapi.org/3.2/o/csv#class-CSV-label-Custom+Header+Converters
+  HEADER_CONVERTERS = {
+    downcase: ->(value) { value.downcase },
+    # Details:
+    #
+    #  Strips leading and trailing whitespace.
+    #  Downcases the header.
+    #  Replaces embedded spaces with underscores.
+    #  Removes non-word characters.
+    #  Makes the string into a Symbol.
+    symbol: ->(value) {
+      value.strip.downcase.tr(" ", "_").gsub(/W+/, "").to_sym
```

Comment:
`W` should be `\W`. Missing spec coverage?

Reply:
You're absolutely right! @olleolleolle, add a test or two for the converters, perhaps just as individual unit tests of those procs.